### PR TITLE
Issue5: 'NODE_ENV' not recognized on Windows

### PR DESCRIPTION
**Pull request by macdonst**
Uses cross_env package to set environment variables on windows, mac and unix.

## Description

Update the package.json file to use cross_env package to set environment variables on windows, mac and unix. Also changed how $NODE_DEBUG_OPTION is passed into `nodemon` to make it cross platform.

## Related Issue

#5 

## Motivation and Context

Fixes windows bug

## How Has This Been Tested?

Tested on my Macbook and Surface Laptop.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
+    "dev": "cross-env NODE_ENV=development PORT=9080 nodemon INSPECT=$NODE_DEBUG_OPTIONS --inspect $INSPECT --watch web-src/resources --watch actions/ --watch manifest.yml ./scripts/parcel.proxy.js",
+    "build:ui": "cross-env NODE_ENV=production REMOTE_ACTIONS=true node scripts/build.ui.js",
+    "build:actions": "cross-env NODE_ENV=production node scripts/build.actions.js",
     "build": "npm run build:ui && npm run build:actions",
-    "deploy:ui": "NODE_ENV=production node scripts/deploy.ui.js",
-    "deploy:actions": "NODE_ENV=production node scripts/deploy.actions.js",
+    "deploy:ui": "cross-env NODE_ENV=production node scripts/deploy.ui.js",
+    "deploy:actions": "cross-env NODE_ENV=production node scripts/deploy.actions.js",
     "deploy": "npm run deploy:actions && npm run deploy:ui",
     "undeploy:ui": "NODE_ENV=production node scripts/undeploy.ui.js",
```
Comment (moritzraho) [inline package.json]: I guess we should add it for the newly merged undeploy cmd sas well

**Review by moritzraho — CHANGES_REQUESTED** [review]
Thanks for providing a PR so quickly @macdonst that's awesome :)
Just need to add the cross env for the newly merged undeploy cmd as well

**Comment by macdonst** [comment]
@moritzraho I rebased my branch with master, made the change and pushed to this PR. Should be up to date now.

**Comment by moritzraho** [comment]
@macdonst thanks a lot ! 
Sorry for the late suggestion but have you considered using a package like: https://github.com/laggingreflex/win-node-env ? (just found this)
This eliminates the need of putting the `cross-env` keyword in front of every ENV var, which  will likely be forgotten by osx and linux devs

**Comment by moritzraho** [comment]
Maybe we could even just say in the readme that win users should run `npm config set script-shell "C:\\Program Files\\git\\bin\\bash.exe"` (from https://stackoverflow.com/a/46006249). Git should be installed anyways to clone this project. Then we are sure there are no problems with the scripts anymore (thinking at `&&`, `||`, .. bash operators)

**Comment by macdonst** [comment]
@moritzraho it's your project I'm just sending a PR that I think resolves the issue. You folks don't have to merge it if you don't want to.

In my opinion you want this starter project to be as frictionless as possible. So if you can support an out of the box solution on Windows without requiring users to install any additional packages or configuring a shell than that is how I would do it.

**Review by moritzraho — APPROVED** [review]

**Comment by moritzraho** [comment]
@macdonst spoke to the team, frictionless is the way to go, I will merge this one now, thanks :)

**Pull request merged**